scopes/preview/ui/component-preview/preview.tsx

```diff
@@ -55,5 +55,5 @@ export function ComponentPreview({
 
   const url = toPreviewUrl(component, previewName, queryParams);
 
-  return <iframe {...rest} ref={iframeRef} style={{ ...style, height }} src={url} />;
+  return <iframe {...rest} ref={iframeRef} style={{ ...style, height: height || 'calc(100vh + 200px)' }} src={url} />;
```

Comment:
this is bad - it is larger than the screen, and it is bad defaults.

--> apply it in the specific consumer